src/nimlsp.nim

```diff
@@ -295,7 +295,8 @@ proc main(ins: Stream | AsyncFile, outs: Stream | AsyncFile) {.multisync.} =
                 addedSuggestions: HashSet[string]
               for suggestion in suggestions:
                 seenLabels.inc suggestion.collapseByIdentifier
-              for suggestion in suggestions:
+              for i in countUp(0,suggestions.len-1):
```

Comment:
This should use `0..suggestions.high` instead

Reply:
Thanks for the tip. still very much learning the nim way of doing things :)